source/js/view.js

```diff
@@ -0,0 +1,209 @@
+let playField = document.getElementById('play_field');
+let rowsInput = document.getElementById('rowsInput');
+let columnsInput = document.getElementById('columnsInput');
+let winLengthInput = document.getElementById('winLengthInput');
+let playerInstancesCollection = document.getElementById('player_instances_collection');
+let cellsCollection = document.getElementsByClassName("cell");
+
+function getAvailableSymbolsList() {
+    return viewModel.getAvailableSymbolsList();
+}
+
+/************ Starting game ************/
+window.onload = function() {
+    let list = viewModel.getPlayersCollection();
+    rowsInput.value = viewModel.getRows();
+    columnsInput.value = viewModel.getColumns();
+    winLengthInput.value = viewModel.getWinLength();
+
+    for (let i = 0; i < list.length; i++) {
+        addPlayerInstance(list[i]);
+        updateSymbolsList();
+    }
+};
+
+function addPlayerInstance(player) {
+    let template = document.getElementById('player_instance_template');
+
+    if ('content' in document.createElement('template')) {
+        playerInstancesCollection.appendChild(template.content.cloneNode(true));
+
+        let playerInstancesList = document.getElementsByClassName('player_instance');
+        let playerInstance = playerInstancesList[playerInstancesList.length - 1];
+        let nameField = playerInstance.querySelector('.form-group .player_name');
+        let symbolField = playerInstance.querySelector('.form-group .player_symbol');
+        let symbolOption = new Option(player.symbol, player.symbol, true, true);
+        let typeField = playerInstance.querySelector('.form-group .player_type');
+
+        playerInstance.setAttribute('id', player.id);
+        nameField.value = player.name;
+        symbolField.appendChild(symbolOption);
+        createTypeList(player.type, typeField);
+    }
+}
+
+function updateSymbolsList() {
+    let symbolsCollection = document.getElementsByClassName('player_symbol');
+    let list = getAvailableSymbolsList();
+
+    for (let k = 0; k < symbolsCollection.length; k++) {
+        let selectedSymbol = symbolsCollection[k].options[symbolsCollection[k].selectedIndex];
+        symbolsCollection[k].options.length = 0;
+
+        if (selectedSymbol) symbolsCollection[k].appendChild(selectedSymbol);
+        for (let i = 0; i < list.length; i++) {
+            let option = new Option(list[i], list[i]);
+            symbolsCollection[k].appendChild(option);
+        }
+    }
+}
+
+function createTypeList(playerType, field) {
+    let playerTypeList = viewModel.getPlayerTypeList();
+
+    for (let type in playerTypeList) {
+        if (playerTypeList.hasOwnProperty(type)) {
+            let selectedType = (playerType === type);
+            let option = new Option(type, type, selectedType, selectedType);
+            field.appendChild(option);
+        }
+    }
+}
+
+function startGame() {
+    let rows = rowsInput.value;
+    let columns = columnsInput.value;
+
+    playField.innerHTML = '';
```

Comment:
This line seems to fit the `createPlayField' function better. You can put it there as a first line of code.